src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String REGEX = ",";
+    private static final int NAME_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int buySum = 0;
+        int supplySum = 0;
+        for (String line: readFile(fromFileName).split(System.lineSeparator())) {
+            String[] row = line.split(REGEX);
+            if (row[NAME_INDEX].equals("buy")) {
+                buySum += Integer.parseInt(row[NUMBER_INDEX]);
+            } else {
+                supplySum += Integer.parseInt(row[NUMBER_INDEX]);
+            }
+        }
+        int result = supplySum - buySum;
```

Comment:
this variable is optional, you can immediately specify it in StringBuilder. for your consideration